time-manager/src/main/java/com/timemanager/exception/IncorrectPassword.java

```diff
@@ -0,0 +1,8 @@
+package com.timemanager.exception;
+
+public class IncorrectPassword extends RuntimeException {
```

Comment:
It will be better if the class which extends Exception contains Exception in this class name.
IncorrectPassword -> IncorrectPasswordException

Reply:
Thank you, I fixed